Dockerfile.base

```diff
@@ -0,0 +1,41 @@
+# syntax = docker/dockerfile:1
+FROM nvcr.io/nvidia/pytorch:22.12-py3
```

Comment:
Is there something restricting us from going to 23.08?

Reply:
Nope! Will test if that works too.